pp.yaml

```diff
@@ -0,0 +1,225 @@
+## User defined edits
```

Comment:
Once the yamls are updated, this will be out of date. 

Also, I think yamls (main and experiment) will be living in a model's repo, not here. Now if we're talking an example yaml, there is currently an example `am5.yaml` and `yaml_include/pp[experiment].yaml` being worked on over here: https://github.com/NOAA-GFDL/fre-cli/tree/99.update-pp-yaml/fre/pp/ppyaml_test (close to the `configure_script_yaml.py`). Thoughts on where example yamls should be?